frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/ticketGroups.component.tsx

```diff
@@ -78,11 +80,11 @@ export const TicketGroups = ({ value, onChange, onBlur }: TicketGroupsProps) =>
 		return highlightedOverride.index;
 	};
 
-	const onDeleteColoredGroup = (index) => {
+	const onDeleteGroup = (type) => (index) => {
 		const newVal = cloneDeep(value);
-		newVal.state.colored.splice(index, 1);
+		newVal.state[type].splice(index, 1);
 		onChange?.(newVal);
-		if (highlightedOverride.index === index) {
+		if (highlightedOverride.index === index && highlightedOverride.type === type) {
 			setHighlightedOverride({ index: -1, type: OverrideType.COLORED });
```

Comment:
I think it could be clearer to just have a clearHighlightedOverride() function for this types of things.